packages/console/src/consts/logs.ts

```diff
@@ -1,54 +1,60 @@
-    'Interaction.ForgotPassword.Identifier.VerificationCode.Create':
-      'Create and send forgot-password verification code',
-    'Interaction.ForgotPassword.Identifier.VerificationCode.Submit':
-      'Submit and verify forgot-password verification code',
-    'Interaction.ForgotPassword.Profile.Create': 'Put new forgot-password interaction profile',
-    'Interaction.ForgotPassword.Profile.Delete': 'Delete forgot-password interaction profile',
-    'Interaction.ForgotPassword.Profile.Update': 'Patch update forgot-password interaction profile',
-    'Interaction.ForgotPassword.Submit': 'Submit forgot-password interaction',
-    'Interaction.ForgotPassword.Update': 'Update forgot-password interaction',
-    'Interaction.Register.Identifier.Password.Submit': undefined,
-    'Interaction.Register.Identifier.Social.Create': undefined,
-    'Interaction.Register.Identifier.Social.Submit': undefined,
-    'Interaction.Register.Identifier.VerificationCode.Create':
-      'Create and send register identifier with verification code',
-    'Interaction.Register.Identifier.VerificationCode.Submit':
-      'Submit and verify register verification code',
-    'Interaction.Register.Profile.Create': 'Put new register interaction profile',
-    'Interaction.Register.Profile.Delete': 'Delete register interaction profile',
-    'Interaction.Register.Profile.Update': 'Patch update register interaction profile',
-    'Interaction.Register.Submit': 'Submit register interaction',
-    'Interaction.Register.Update': 'Update register interaction',
-    'Interaction.SignIn.Identifier.Password.Submit': 'Submit sign-in identifier with password',
-    'Interaction.SignIn.Identifier.Social.Create': 'Create social sign-in authorization-url',
-    'Interaction.SignIn.Identifier.Social.Submit': 'Authenticate and submit social identifier',
-    'Interaction.SignIn.Identifier.VerificationCode.Create':
-      'Create and send sign-in verification code',
-    'Interaction.SignIn.Identifier.VerificationCode.Submit':
-      'Submit and verify sign-in identifier with verification code',
-    'Interaction.SignIn.Profile.Create': 'Put new sign-in interaction profile',
-    'Interaction.SignIn.Profile.Delete': 'Delete sign-in interaction profile',
-    'Interaction.SignIn.Profile.Update': 'Patch Update sign-in interaction profile',
-    'Interaction.SignIn.Submit': 'Submit sign-in interaction',
-    'Interaction.SignIn.Update': 'Update sign-in interaction',
-    'TriggerHook.PostRegister': undefined,
-    'TriggerHook.PostResetPassword': undefined,
-    'TriggerHook.PostSignIn': undefined,
-    RevokeToken: undefined,
-    Unknown: undefined,
-  });
+/**
+ * Filter out webhook logs from the audit log list.
+ * Hence should not allow user to check webhook logs by selecting their event/log types.
+ */
+type AuditLogKeys = Exclude<
+  LogKey,
+  'TriggerHook.PostRegister' | 'TriggerHook.PostSignIn' | 'TriggerHook.PostResetPassword'
+>;
```

Comment:
should define the type in the same place as the original LogKey?  Webhook logs vs AuditLogs is more like data model level definition instead of client-side definition